components/send/LedgerSending.tsx

```diff
@@ -60,6 +62,12 @@ export default function LedgerSending({
       }
 
       setOutTxs(transactionResponse.txIds);
+      // Don't await, analytics should not crash the app
+      emitFirstTransaction({
+        amount: sompiToKaspaString(amount),
```

Comment:
https://github.com/forbole/kastle/blob/0e62cf55d2998aad907bab30f6e3fbdefb74e26e/components/send/LedgerSending.tsx#L56
The amount is already kaspa not sompi.

Reply:
3a51dce50cebfcd422b70d2b53607775af044247